includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -1,21 +1,185 @@
 <?php
 
 class DesignSystemApiController extends WikiaApiController {
+	private $wikiId;
+	private $lang;
+
+	private $data = [
+		'brand-logo' => [
+			'type' => 'brand-logo',
+			'href' => '#',
+			'elements' => [
```

Comment:
Can we keep all structure-related comments in one place? : https://docs.google.com/document/d/1oz4IWS_NB11AnL0GS09dWpw1sppCiolhpRR-DVFImwk/edit#